bot.py

```diff
@@ -552,7 +553,13 @@ async def lab_hours_reminder():
             if special_hours:
                 description += f'\n**Special Lab Hours!**:\n{special_hours}'
             embed = discord.Embed(title=title, description=description, color=color)
-            await lab_channel.send(embed=embed)
+
+            if lab_channel:
+                # Send differently if plaintext or rich embed
+                if type(embed) is str:
```

Comment:
This seems unnecessary since embed is set as a discord embed a couple lines above